django_project/frontend/views/users.py

```diff
@@ -315,8 +324,25 @@ def get_organisation_users(self, request):
                     "assigned_as": assigned_as,
                     "joined": False
                 }
-            if not user.user == request.user:
-                organisation_users.append(object_to_save)
+
+            organisation_users.append(object_to_save)
+
+        organisation_reps_list = list(
+            OrganisationRepresentative.objects.filter(
+                organisation_id=get_current_organisation_id(request.user)
+            ).exclude(
+                user__in=[uid['id'] for uid in organisation_users]
```

Comment:
@zamuzakki I think we should find the OrganisationRepresentative first, then query the OrganisationUser excluding the managers. Because if we find manager in the latter, the user_id already exists in organisation_users.